obj/objects.h

```diff
@@ -68,9 +69,9 @@ namespace objects
     class EnergyCollector : public Sphere
     {
     public:
-        EnergyCollector(const core::Vec3 &origin, double energy) : _id(id), _energy(energy)
+        EnergyCollector(const core::Vec3 &origin, double energy = 0) : _energy(energy) // TODO: delete argument for energy and assing 0 to energy by default
         {
-            setRadius(constants::kPi * constants::kSimulationRadius / constants::kPopulation);
+            setRadius(constants::kPi * (constants::kSimulationRadius) / constants::kPopulation); // ( 2 *  pi * R ) / 2 is for dividing whole sphereWall to qual distances
```

Comment:
Comments are written before the line they are supposed to comment on. Unless they are one or two words long.

A proper TODO comment would be to also remove the dependency on the constants.

Reply:
noted